src/main/java/com/hr/controller/HireController.java

```diff
@@ -27,7 +27,7 @@ public String addEmployee(@ModelAttribute(value="employee") Employee employee, M
         }
 
         model.addAttribute("employee", DUMMY);
-        model.addAttribute("message", "hiring failed because you add nothing");
+        model.addAttribute("message", "hiring failed since employee is not in our repository");
```

Comment:
This technical language should not be leaking to the user.

Reply:
Right, I will fix that